Sources/MessagingPush/UserNotificationsFramework/Wrappers.swift

```diff
@@ -0,0 +1,83 @@
+import Foundation
+import UserNotifications
+
+/**
+ The SDK's code is abstracted away from the iOS `UserNotifications` framework.
+
+ This file contains code that runs in production. Code that makes the `UserNotifications` data types
+ conform to all of the SDK's abstracted data types.
+
+ All of these wrappers should be small and simple. Their only job is to convert data types between SDK's abstracted data types and `UserNotifications` data types.
+ */
```

Comment:
I thought it might be annoying to have 1 file for every class in this file. Since all of them are simple wrappers. 

Happy to split into multiple files if others prefer.